aten/src/ATen/native/FractionalMaxPool2d.cpp

```diff
@@ -134,8 +134,9 @@ static std::vector<int> fractional_max_pool2d_generate_intervals(
         static_cast<int>((i + sample) * alpha) - static_cast<int>(sample * alpha);
     }
   }
-  sequence[outputSize - 1] = inputSize - poolSize;
-
+  if (outputSize >= 1) {
```

Comment:
nit: `outputSize > 0` to be consistent with the other instance

Reply:
Good catch! thanks :)